curriculum/challenges/english/20-upcoming-python/learn-lambda-functions-by-building-an-expense-tracker/6582507654b3ed5712341382.md

```diff
@@ -0,0 +1,75 @@
+---
+id: 6582507654b3ed5712341382
+title: Step 29
+challengeType: 20
+dashedName: step-29
+---
+
+# --description--
+
+You are going to use conditional statements to check the user's choice. If the choice is `1`, it means the user wants to add an expense.
+
+Still in the `while` loop, under the `choice` variable, write an `if` statement to check if `choice` equals the string `1`. If it's true, it will be the starting point for adding a new expense.
```

Comment:
```suggestion
Still in the `while` loop, under the `choice` variable, write an `if` statement to check if `choice` equals the string `1`. If it's true, this will be the starting point for adding a new expense.
```